website/src/pages/docs/grid.mdx

```diff
@@ -0,0 +1,64 @@
+---
+menu: Guides
+title: Grid
+order: 1
+---
+
+# Grid
+
+Quickly style your app with column and rows. 
+
+## Create a grid
+
+- Give the property `row` to a Box to automatically create a grid.
+- Give the property `col` to a Box to create a column.
+- Fill up the grid by giving a value comprised between 0 and 1 to each `col`.
+- By default, a `col` width is set to 1 and thus takes the entire space.
```

Comment:
Nop, by default a `col` width is set to `true`, it is equivalent to a "flex-grow: 1".

Reply:
Yes, but does the user needs to know the col's nuts and bolts? In the end, the latter only wants to know what he has to enter to make the grid work.